tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -1051,33 +1048,17 @@ TEST_F( HVACFixture, VRFTest_SysCurve ) {
 
 	}
 
-TEST_F( HVACFixture, VRFTest_SysCurve_WaterCooled ) {
-
-		static std::string const RoutineName( "VRFTest_WaterCooled" );
+	TEST_F( HVACFixture, VRFTest_SysCurve_GetInputFailers ) {
+		// Author: R. Raustad, FSEC
+		
```

Comment:
The unit tests keep getting stomped on. This shouldn't hold up the commit. I can come around and fix once the VRF bugs are checked in.